detox/src/logger/utils/BunyanLogger.js

```diff
@@ -41,13 +41,14 @@ class BunyanLogger {
       streamOptions.out = new PassThrough().pipe(process.stderr);
     }
 
-    this._debugStream = {
+    this._bunyan.addStream({
```

Comment:
Turned out, that `addStream` does not add the given object as-is. That's why you can't use it as a reference to remove it from the internal `_streams` array of the Bunyan logger.